test/unit/app/browser/reducers/ledgerReducerTest.js

```diff
@@ -969,4 +971,12 @@ describe('ledgerReducer unit tests', function () {
       assert.deepEqual(result.toJS(), expectedState.toJS())
     })
   })
+
+  describe('APP_ON_PROMOTION_CLICK', function () {
+
+  })
+
+  describe('APP_ON_CAPTCHA_RESPONSE', function () {
+
+  })
```

Comment:
Do we need tests on these two?

Reply:
lol I added them but then forgot to actually add them 😄 thx

Reply:
tests added